.github/workflows/ios.yml

```diff
@@ -7,10 +7,10 @@ jobs:
     strategy:
       matrix:
         # macos-latest (ATM macos-15) runs on Apple Silicon,
-        # macos-13 runs on Intel
-        runs_on: [macos-latest, macos-13]
+        # macos-15-intel runs on Intel
+        runs_on: [macos-latest, macos-15-intel]
```

Comment:
How about using explicit naming for both so we know what we're talking about?

Reply:
I mean, why not `macos-15` rather than `macos-latest` so we're explicit about what version we're trying to run.
also any plan on running `macos-14`?
Also I like that the label `macos-15-intel` is already explicit about the architecture, that way we can remove the comment above it